public/app/plugins/datasource/influxdb/query_ctrl.ts

```diff
@@ -8,6 +8,9 @@ import { TemplateSrv } from '@grafana/runtime';
 import { InfluxQuery } from './types';
 import InfluxDatasource from './datasource';
 
+const GET_MEASUREMENTS_TIMEOUT = 5 * 60 * 1000;
```

Comment:
this delay was chosen to be very long, so not to interfere with normal behavior. in theory, this could cause problems if getting the measurement-list from influxdb takes more than 5 minutes, but i don't think that is realistic.